ecs/run-task/main.go

```diff
@@ -21,10 +25,61 @@ func main() {
 
 	ecsClient := ecs.New(session, conf)
 
+	containerOverrides, overridesErr := resolveContainerOverrides(envVars)
+	common.FatalOnError(overridesErr)
+
 	_, err := ecsClient.RunTask(&ecs.RunTaskInput{
 		TaskDefinition: taskDefinition,
 		Cluster:        cluster,
 		Count:          aws.Int64(1),
+		Overrides: &ecs.TaskOverride{
+			ContainerOverrides: containerOverrides,
+		},
 	})
+
 	common.FatalOnError(err)
 }
+
+func resolveContainerOverrides(environmentVariables *map[string]string) ([]*ecs.ContainerOverride, error) {
+	containerOverrides := []*ecs.ContainerOverride{}
+	containerOverridesMap := map[string]*ecs.ContainerOverride{}
+
+	if len(*environmentVariables) > 0 {
+		for k, v := range *environmentVariables {
+			containerName := k
+
+			split := strings.Split(v, "=")
```

Comment:
isn't the splitting on `=` done by the StringMap already? Shouldn't that be splitting on the `:` in `k` instead of `v` then get `split[0]` = container name, `split[1]` = variable name and v = variable value?